src/app/system/surveys/surveys.component.scss

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
Take reference from `Accounting` section for implementation and remove all unnecessary classes.